server/athenian/api/openapi/openapi.yaml

```diff
@@ -5244,6 +5244,7 @@ components:
         - jira-bug-raised
         granularities:
         - all
+        exclude_inactive: true
```

Comment:
I think this is also missing in other endpoints examples like filter/pull_requests if I recall correctly (I can not check as swagger is not available for staging)

Reply:
ok, I'll check :+1:

Reply:
You can check in prod btw.